### PR TITLE
Untrack

### PR DESCRIPTION
**Pull request by amitgilad3**

### CODE REVIEW

File: src/api/consumer/lib/untrack.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+
+export default async function untrack(componentIds: string[]): Promise<Object> {
+
+  const untrackedComponents = [];
+  const consumer: Consumer = await loadConsumer();
+  const bitMap = await BitMap.load(consumer.getPath());
+  const componentsList = new ComponentsList(consumer);
+  const newComponents = await componentsList.listNewComponents(true);
+  const stagedComponents = await componentsList.listExportPendingComponents();
```
Comment (GiladShoham) [inline src/api/consumer/lib/untrack.js]: get the list export / list modified is expensive operations and you don't really need them.
If a component is not missing and not in the new list it should just be in the unRemovable

File: src/api/consumer/lib/untrack.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  const modifiedComponent = await componentsList.listModifiedComponents(true);
```
Comment (GiladShoham) [inline src/api/consumer/lib/untrack.js]: I think the logic here should be simpler
If component ids is empty just run over the newComps and remove them.
If it's not empty, run over them, and for each one:
check if is missing, if not, check if it's in the new list if yes remove, if not add to the unRemovabale.

File: src/bit-id/bit-id.js [hunk 3]
```diff
@@ -131,6 +131,36 @@ export default class BitId {
     throw new InvalidBitId();
   }
 
+  static isValidBitId(id: string) {
```
Comment (GiladShoham) [inline src/bit-id/bit-id.js]: This is not related to this feature (also i don't see you use it). please make a different PR for this.

Comment (amitgilad3) [reply src/bit-id/bit-id.js]: yes ,my mistake , i removed this
